spec/features/officing_spec.rb

```diff
@@ -169,8 +171,10 @@
       expect(page).to have_content("Here you can validate user documents and store voting results")
 
       visit new_officing_residence_path
-      select 'DNI', from: 'residence_document_type'
-      fill_in 'residence_document_number', with: "12345678Y"
+      expect(page).to have_selector('#residence_document_type')
+
+      select 'Passport', from: 'residence_document_type'
+      fill_in 'residence_document_number', with: "12345678A"
```

Comment:
Why is necessary to change the residence document number value?

Reply:
That's the result of the cherry-pick. I just changed it back after seeing the results on travis

Reply:
Reviewing the diff is the second thing (after reading the PR description) that a PR reviewer will do. Authors should always review PR after creating to catch this simple mistakes, reviewers have 100x PR's pending